USER_GUIDE.md

```diff
@@ -221,32 +223,39 @@ func main() {
 
 #### AWS SDK V2
 
+AWS SDK V2 for Go can be used for various service options. To create a client, please see the example below:
```

Comment:
I don't think we need to be polite in docs, would remove "please".

I also prefer succinct versions, so "Use the AWS SDK v2 for Go to authenticate with Amazon OpenSearch service."